<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelSetting</class>
 <widget class="QDialog" name="ChannelSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>导联设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_lineedit_ch_name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>导联名称</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="m_pushButton_add_sig">
       <property name="text">
        <string>添加信号</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QChartView" name="m_chartview" native="true">
       <property name="minimumSize">
        <size>
         <width>340</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>99999</width>
         <height>99999</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="QTableWidget" name="m_tableWidget_sig">
       <column>
        <property name="text">
         <string>信号描述</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
